command/sub8_launch/config/online_bagger.yaml

```diff
@@ -0,0 +1,12 @@
+# list an arbitrary set of topics to  subscribe to and stream:
+
+stream_time: 60  # seconds
+
+topics: [["/odom",                          ],
```

Comment:
This list is unnecessary now that it pulls from the ```bag_*``` environment variables. The bagging topics will now be version controlled through the bag aliases file.